packaging/installer/netdata-updater.sh

```diff
@@ -396,17 +554,26 @@ done
 # But only we're not a controlling terminal (tty)
 # Randomly sleep between 1s and 60m
 if [ ! -t 1 ] && [ -z "${NETDATA_NOT_RUNNING_FROM_CRON}" ]; then
-    sleep $(((RANDOM % 3600) + 1))
+    rnd="$(awk '
+      BEGIN { srand()
+              printf("%d\n", 3600 * rand())
+      }')"
+    sleep $(((rnd % 3600) + 1))
 fi
 
 # shellcheck source=/dev/null
-source "${ENVIRONMENT_FILE}" || exit 1
+. "${ENVIRONMENT_FILE}" || exit 1
+
+if [ -f "$(dirname "${ENVIRONMENT_FILE}")/.install-type" ]; then
+  # shellcheck source=/dev/null
+  . "$(dirname "${ENVIRONMENT_FILE}")/.install-type" || exit 1
+fi
```

Comment:
We could have an `INSTALL_TYPE_FILE` variable right above `safe_sha256sum`'s definition and just source it.